internal/global/common/bleed.go

```diff
@@ -99,3 +99,13 @@ func breakBleedPhase1(mod *modifier.Instance) {
 		UseSnapshot:  true,
 	})
 }
+
+// Custom event trigger for bleed dots
+func (B BleedState) TriggerEvent(mod *modifier.Instance) {
```

Comment:
nit: might not be clear to users what `TriggerEvent` means when reading character implementations (though not against the name)

Thinking about implementations, such as Sampo E4 or Kafka, these would also need to take in a `Ratio` which gets multiplied against the original damage percentage of the mod (IE: Sampo E4 is 4%). In the game implementation, this would be that `DOT_TriggerRatio`: https://damnae.github.io/stargazer/#/modifier-MCommon_Element_Electric/from/avatar-1005/@59d64be43a1da285cf22ba9be5ed90ef2b23f857